configure

```diff
@@ -78,10 +78,21 @@ done
 
 
 # Find out where the CUDA toolkit is installed
+OSNAME=`uname -s`
+
 while true; do
   # Configure the Cuda SDK version to use.
   if [ -z "$TF_CUDA_VERSION" ]; then
-    read -p "Please specify the Cuda SDK version you want to use, e.g. 7.0. [Leave empty to use system default]: " TF_CUDA_VERSION
+    if [ "$OSNAME" == "Linux" ]; then
+      default_cuda_version='7.0'
```

Comment:
Oh, sounds like Linux has caught up;) When I created the pull request,  master was using 7.0 with Linux, but OS X clang would only work with 7.5.  I'll make both default to 7.5.